src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -137,22 +137,13 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         noExceptionThrown()
     }
 
-    @PendingFeature
     def "task can be up-to-date"() {
         given:
         buildFile << imageCreation()
-        buildFile << """
-            task verifyImageId {
-                dependsOn buildImage
-                doLast {
-                    assert buildImage.imageId.isPresent()
-                }
-            }
-        """
-        File imageIdFile = new File(projectDir, 'build/.docker/buildImage-imageId.txt')
+        File imageIdFile = new File(projectDir, 'build/.docker/:buildImage-imageId.txt')
```

Comment:
I believe there also was an issue with using the colon character on Windows machines.